common/src/main/java/com/microsoft/identity/common/internal/commands/DeviceCodeFlowCommand.java

```diff
@@ -72,7 +80,14 @@ public AcquireTokenResult execute() throws Exception {
         );
 
         // Call acquireDeviceCodeFlowToken to get token result (Part 2 of DCF)
-        return controller.acquireDeviceCodeFlowToken(authorizationResult, commandParameters);
+        final AcquireTokenResult tokenResult = controller.acquireDeviceCodeFlowToken(authorizationResult, commandParameters);
+
+        Logger.verbose(
+                TAG + methodName,
+                "Device Code Flow command exiting with token..."
```

Comment:
> with token

Are we certain about this?

Reply:
I think this is a safe assumption. If the token request had failed, a service exception would've been thrown and this log wouldn't have been created.